src/lib/y2network/network_manager/connection_config_writers/base.rb

```diff
@@ -64,7 +64,7 @@ def write(conn, opts = {})
         #
         # @param routes [<Array<Y2Network::Route>] routes associated with the connection
         def configure_routes(routes)
-          routes.select(&:default?).each { |r| configure_gateway(r) }
+          routes.select(&:default?).each { |r| configure_gateway(r) if r.gateway }
```

Comment:
Well ... it solves the issue by not raising the internal error. But question is if we want to get into this situation. From logs 
```2022-09-28 05:19:41 <1> install(3712) [Ruby] dialogs/route.rb(initialize):38 route dialog with route: #<Y2Network::Route:0x00005574bc318518 @to=:default, @interface=nil, @gateway=nil, @options=""> and devices ...``` 
so it seems that default route was created without an ip. Does it make sense to allow it?

Reply:
it is allowed by the interface and also by wicked which permits for example a default just with device, so, once that is allowed in routing config and interface then yes, it makes sense but not when it is translated from wicked to NM which is the current issue.

Reply:
Yeah device route(s) are fine, but even ```@interface``` is nil here so it still looks weird to me. But i didn't check code in details.

Reply:
if there is no interface then there should be probably not associated to the connection related to the interface... as that route do not makes any sense we could add a validation in the route dialog to prevent that.

Reply:
yep ... that is exactly what i tried to point out. So, what to do now?

Reply:
> Yeah device route(s) are fine, but even `@interface` is nil here so it still looks weird to me. But i didn't check code in details.

Confirmed this situation is not crashing, so the issue really happen when there was a specific device declared in the route.

Reply:
ok then